### ISSUE TITLE
Notice (8): Undefined variable: _icons [APP\Vendor\croogo\croogo\Gallery\View\Albums\admin_upload.ctp, line 10]

### ISSUE THREAD

**Issue by amjo** [xintesa/Gallery#13]
Hello there,

I successfully installed Imagine, Assets, and finally Gallery. Basically I'm facing the same issue 100% as the issue reported here: https://github.com/xintesa/Gallery/issues/12 - i am able to create an album, but unable to upload photos.

Furthermore, even before i get to try uploading, i get 

> Notice (8): Undefined variable: _icons [APP\Vendor\croogo\croogo\Gallery\View\Albums\admin_upload.ctp, line 10]

this is when i get to [site]/admin/gallery/albums/upload/1
![capture](https://cloud.githubusercontent.com/assets/441767/15431167/46ab9200-1eb1-11e6-868f-8d3789342724.JPG)

for the time being, i replaced $_icons['delete/update/move-up/move-down']; with $this->Theme->getIcon('delete/update/move-up/move-down'); but im guessing I shouldn't do this and the $_icon[] should be defined somehow?

**Comment by Einzelnman** [comment]
I think you using the gallery plugin 3.0.
This bug is fixed in 3.1, check this out: https://github.com/xintesa/Gallery/commit/d39dbc8e37ce7f091612620643f17480c85f9098
but the 3.1 is incompatible with croogo 2.2.3, so we need to waiting a new rls or we fix it.
OR create you own plugin, its a good practice. :)

**Comment by amjo** [comment]
I have created my own plugin years ago, and have been maintaining it ever since. of course, it's not a logical solution to develop an entire plugin from scratch jsut because of a missing variable. i found a work-around for the above - at least (intended for future readers)

this was the least of my concerns as issue #12 is my main problem now. instead i rolled back to using 2.1 which is compatible with Croogo 1.5 and 2.2.3. :)

thanks anyway

**Comment by Einzelnman** [comment]
yes yes, the 2.1 is working fine, because it dont use the Assetts plugin. :)
